scripts/ci-scripts/start-server.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+SRCROOT=$(cd "$(dirname "$0")/../.." && pwd)
+
+echo "test.sh: starting stitch"
```

Comment:
```suggestion
echo "${0}: starting stitch"
```
The name of this script isn't `test.sh`.
Maybe we should remove the prefix, though.  It's not used in the logging messages below..

Reply:
I'm just removing this script.  It is replaced by `start-sync-server.sh`